Sources/InstantSearchSwiftUI/View/HitsList.swift

```diff
@@ -68,42 +68,83 @@
     }
   }
 
-  #if os(iOS)
-    @available(iOS 13.0, tvOS 13.0, watchOS 7.0, *)
-    struct HitsView_Previews: PreviewProvider {
-      static var previews: some View {
-        let hitsController: HitsObservableController<String> = .init()
-        NavigationView {
-          HitsList(hitsController) { string, _ in
-            VStack {
-              HStack {
-                Text(string ?? "---")
-                  .frame(maxWidth: .infinity, minHeight: 30, maxHeight: .infinity, alignment: .leading)
-                  .padding(.horizontal, 16)
-              }
-              Divider()
+  @available(iOS 13.0, tvOS 13.0, watchOS 7.0, *)
+  struct HitsView_Previews: PreviewProvider {
+    
+    struct PreviewRecord<Value: Codable>: Codable {
+      let objectID: ObjectID
+      let value: Value
+      
+      init(_ value: Value, objectID: ObjectID = ObjectID(rawValue: UUID().uuidString)) {
+        self.value = value
+        self.objectID = objectID
+      }
+      
+      static func withValue(_ value: Value) -> Self {
+        .init(value)
+      }
+    }
+    
+    static let rawHits: Data = """
```

Comment:
What's the goal of this change? I purposely used `String` as a simplest type to simplify testing as much as possible.